packages/multichain-network-controller/package.json

```diff
@@ -68,7 +68,8 @@
     "typescript": "~5.2.2"
   },
   "peerDependencies": {
-    "@metamask/network-controller": "^22.2.1"
+    "@metamask/accounts-controller": "^23.0.0",
+    "@metamask/network-controller": "^22.0.0"
```

Comment:
We only use the current major version for that controller, we don't need a specific feature of a specific version for the moment.